kong/status/ready.lua

```diff
@@ -102,6 +103,16 @@ end
 return {
   ["/status/ready"] = {
     GET = function(self, dao, helpers)
+      local unready = kong_shm:get(KONG_STATUS_UNREADY)
+      if unready == nil then
+        kong_shm:set(KONG_STATUS_UNREADY, false)
+      end
+
+      if unready == true then
+        ngx_log(ngx_DEBUG, "not ready for proxying: unready")
```

Comment:
```suggestion
```

This log line seems ineffective, and could possibly flood the error log if the `/status/ready` is configured as healthcheck endpoint

Reply:
done